README.md

```diff
@@ -10,7 +10,7 @@ hosting [fonts releases](https://github.com/parlr/hanzi-pinyin-font/releases) |
 
 | ruby position-direction | preview | state | comments
 | --- | :---: | :---: | --- |
-| top | ![top](images/annotation-top.png)  | **✔** | [TTF](Hanzi-Pinyin-Font.top.ttf), [WOFF2](Hanzi-Pinyin-Font.top.woff2)
+| top | ![top](images/annotation-top.png)  | **✔** | [TTF](Hanzi-Pinyin-Font.top.ttf), [WOFF2](Hanzi-Pinyin-Font.top.woff2), [HWT](Hanzi-Pinyin-Font.top.hwt)
```

Comment:
Thanks for the effort!
What is the base font you used? Does its license allow for modification?
I'm using [open-source fonts](https://github.com/parlr/ruby-font-creator?tab=readme-ov-file#font) as proprietary fonts might not allow such use.